analyzers/src/SonarAnalyzer.CFG/CfgSerializer/CfgSerializer.RoslynCfgWalker.cs

```diff
@@ -33,14 +34,16 @@ public partial class CfgSerializer
         private class RoslynCfgWalker
         {
             private readonly DotWriter writer;
-            private readonly HashSet<BasicBlock> visited = new HashSet<BasicBlock>();
+            private readonly HashSet<BasicBlock> visited = new();
             private readonly RoslynCfgIdProvider cfgIdProvider;
+            private readonly CancellationToken cancellationToken;
```

Comment:
All changes in this file should be reverted. It doesn't make sense to pollute this file with cancellation because it is used only for debugging purposes.
Use `default` instead.